openapi-spec/swagger.yaml

```diff
@@ -708,6 +708,111 @@ paths:
           description: The resource does not exist.
         '500':
           $ref: '#/responses/HTTPStatus500'
+  '/v1beta/{projectId}/block/groups':
```

Comment:
I think `groups` declared here is ambiguous, because it would be some conflicts when adding snapshot group. So some modification (such as `volumeGroups`) need to made here.